dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -729,6 +730,42 @@ def test_correct_total_sample_size_and_counts_and_mutability(self):
         self.assertEqual(col_one_len, len(data['NAME']))
         self.assertEqual(col_two_len, len(data['VALUE']))
 
+    def test_save_and_load(self):
```

Comment:
These tests should probably be under profiler builder rather than a specifically for testing nulls

Reply:
Moved to TestProfiler